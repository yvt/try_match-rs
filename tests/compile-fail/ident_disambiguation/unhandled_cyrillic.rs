fn main() {
    try_match::try_match!(42, привет);
}
