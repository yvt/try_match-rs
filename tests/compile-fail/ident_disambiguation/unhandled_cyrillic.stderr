error: ambiguous identifier pattern
 --> tests/compile-fail/ident_disambiguation/unhandled_cyrillic.rs:2:31
  |
2 |     try_match::try_match!(42, привет);
  |                               ^^^^^^
